src/requirements.txt

```diff
@@ -2,3 +2,5 @@ Flask==1.0.2
 Flask-Markdown==0.3
 flask-talisman
 gunicorn==19.9.0
+pytest
+pytest-watch
```

Comment:
Could you add a section to src/README with instructions for running the tests?

Reply:
Done! I kept it really brief, I hope it's sufficient?